src/arctica-greeter.vala

```diff
@@ -1713,4 +1730,27 @@ public class DBusServer : Object
         debug ("High-Contrast mode toggled (new state: %b), refreshing OSK, as well.", bActive);
         ToggleOnBoard (AGSettings.get_boolean (AGSettings.KEY_ONSCREEN_KEYBOARD));
     }
+
+    public void ToggleMagnifier (bool bActive, string sApplication) throws GLib.DBusError, GLib.IOError
```

Comment:
I don't like the idea of being able to inject any magnifier application here. In fact, _ANY_ application could be launched via this DBus interface.

Personally, we should pick one magnifier application and hard-code it here (or configure the executable in gsettings of Arctica Greeter. But this looks like the DBus client can launch $EVERYTHING.

Or do I get this code portion wrong?

Reply:
I originally had magnus hardcoded, but later changed the D-Bus params. Reverted.